decompiler/level_extractor/extract_level.cpp

```diff
@@ -361,14 +360,12 @@ void extract_from_level(const ObjectFileDB& db,
 
   if (config.rip_levels) {
     auto back_file_path = file_util::get_jak_project_dir() / "glb_out" /
-                          game_version_names[config.game_version] /
+                          game_version_names[config.game_version] / level_data.level_name /
                           fmt::format("{}-background.glb", level_data.level_name);
     file_util::create_dir_if_needed_for_file(back_file_path);
     save_level_background_as_gltf(level_data, back_file_path);
     auto fore_file_path = file_util::get_jak_project_dir() / "glb_out" /
-                          game_version_names[config.game_version] /
-                          fmt::format("{}-foreground.glb", level_data.level_name);
-    file_util::create_dir_if_needed_for_file(fore_file_path);
+                          game_version_names[config.game_version] / level_data.level_name;
```

Comment:
same here, explicitly ensure the directory you are about to save a file to, exists.